### PR TITLE
ATR-250: added help link uri to github

### PR DESCRIPTION
**Pull request by SENya1990**
Merge after the documentation is merged

### CODE REVIEW

**Comment by SENya1990** [comment]
@dropsonic note that it is merged to master. Should I merge it to dev instead?

File: src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs [hunk 1]
```diff
@@ -10,15 +10,19 @@ public enum Category
 
 	public static class Descriptors
 	{
+		private const string DocumentationLinkPrefix = @"https://github.com/Acumatica/Acuminator/blob/dev/docs/diagnostics";
```
Comment (dropsonic) [inline src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs]: Url should be `https://github.com/Acumatica/Acuminator/docs/diagnostics`

**Comment by dropsonic** [comment]
@SENya1990 it is ok to merge it to the master

File: src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs (hunk 1 above)
Comment (SENya1990) [reply src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs]: fixed

**Pull request merged**